  INFO root [2019-09-03 21:24:27,088] - 
  INFO root [2019-09-03 21:24:27,099] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-03 21:24:27,688] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-03 21:24:27,688] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-03 21:24:27,688] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-03 21:24:27,849] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:52644 )
  INFO root [2019-09-03 21:24:27,849] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:52644, SOMAXCONN=5.
  INFO root [2019-09-03 21:24:27,849] - server-configs:
  INFO root [2019-09-03 21:24:27,849] - 	internalTcpAddr : 192.168.56.1:52644
  INFO root [2019-09-03 21:24:27,849] - 	componentID : 5000
  WARN root [2019-09-03 21:24:27,849] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-03 21:24:29,479] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 2082. PID: 8916 ----
  INFO root [2019-09-03 21:24:29,479] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-03 21:24:29,519] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-03 21:24:29,519] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-03 21:24:30,569] - ---- baseappmgr is running ----
 DEBUG root [2019-09-03 21:24:30,569] - Components::process(): Request for the process of identity...
  WARN root [2019-09-03 21:24:35,319] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20286, errorno=10048
  INFO root [2019-09-03 21:24:39,489] - Components::findComponents: find logger(1)...
  INFO root [2019-09-03 21:24:39,609] - Components::findComponents: found logger, addr:192.168.56.1:52648
  INFO root [2019-09-03 21:24:39,673] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-03 21:24:39,673] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-03 21:24:39,759] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-03 21:24:40,889] - Forwarding logs to logger[192.168.56.1:52648/0/0/0]...
 ERROR root [2019-09-03 21:36:03,748] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=270510430637, lastReceivedTime=270504686445, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-03 21:36:03,748] - The following logs sent to logger failed:
  WARN root [2019-09-03 21:36:03,748] - ==> [2019-09-03 21:36:03 751] TCPPacketReceiver::processPendingEvents(192.168.56.1:52648): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-03 21:36:03,748] - ==> [2019-09-03 21:36:03 752] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=270510430637, lastReceivedTime=270504686445, inactivityExceptionPeriod=134887628)
  WARN root [2019-09-03 21:36:03,868] - TCPPacketReceiver::processPendingEvents(192.168.56.1:52649): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-03 21:36:03,868] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=270510708157, lastReceivedTime=270506576133, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-03 21:36:03,879] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-03 21:36:03,879] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-03 21:36:03,879] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-03 21:36:03,908] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-03 21:36:05,612] - 
  INFO root [2019-09-03 21:36:05,612] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-03 21:36:05,612] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-03 21:36:05,612] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-03 21:36:05,612] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-03 21:36:05,642] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:52851 )
  INFO root [2019-09-03 21:36:05,642] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:52851, SOMAXCONN=5.
  INFO root [2019-09-03 21:36:05,642] - server-configs:
  INFO root [2019-09-03 21:36:05,642] - 	internalTcpAddr : 192.168.56.1:52851
  INFO root [2019-09-03 21:36:05,642] - 	componentID : 5000
  WARN root [2019-09-03 21:36:05,642] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-03 21:36:07,151] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 2082. PID: 13684 ----
  INFO root [2019-09-03 21:36:07,151] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-03 21:36:07,151] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-03 21:36:07,151] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-03 21:36:07,568] - ---- baseappmgr is running ----
 DEBUG root [2019-09-03 21:36:07,570] - Components::process(): Request for the process of identity...
  INFO root [2019-09-03 21:36:14,918] - Components::findComponents: find logger(1)...
  WARN root [2019-09-03 21:36:15,538] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20993, errorno=10048
  INFO root [2019-09-03 21:36:16,558] - Components::findComponents: find logger(2)...
  WARN root [2019-09-03 21:36:17,178] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20349, errorno=10048
  INFO root [2019-09-03 21:36:18,198] - Components::findComponents: find logger(3)...
  INFO root [2019-09-03 21:36:18,218] - Components::findComponents: found logger, addr:192.168.56.1:52850
  INFO root [2019-09-03 21:36:18,218] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-03 21:36:18,218] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-03 21:36:18,218] - Components::findComponents: find dbmgr(4)...
  INFO root [2019-09-03 21:36:18,338] - Components::findComponents: found dbmgr, addr:192.168.56.1:52855
  INFO root [2019-09-03 21:36:18,342] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-03 21:36:18,342] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-03 21:36:18,342] - Components::findComponents: found dbmgr, addr:192.168.56.1:52855
  WARN root [2019-09-03 21:36:19,357] - Forwarding logs to logger[192.168.56.1:52850/0/0/0]...
 ERROR root [2019-09-03 21:50:26,960] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=272452675002, lastReceivedTime=272404609776, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-03 21:50:26,975] - The following logs sent to logger failed:
  WARN root [2019-09-03 21:50:26,975] - ==> [2019-09-03 21:50:26 967] TCPPacketReceiver::processPendingEvents(192.168.56.1:52850): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-03 21:50:26,975] - ==> [2019-09-03 21:50:26 971] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=272452675002, lastReceivedTime=272404609776, inactivityExceptionPeriod=134887628)
  WARN root [2019-09-03 21:50:27,069] - TCPPacketReceiver::processPendingEvents(192.168.56.1:52855): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-03 21:50:27,069] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=272452925956, lastReceivedTime=272404835903, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-03 21:50:27,084] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-03 21:50:27,084] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-03 21:50:27,084] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-03 21:50:27,084] - ThreadPool::destroy(): starting size 2.
